### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by commanderjsoc**
## Description

Added spaces and supporting text to make the readme more readable

## Motivation and Context

I like things to look nice and professional.

## How Has This Been Tested?

The readme still can be opened in markdown

## Screenshots (if appropriate):
pic

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,4 @@
-# PokemonGo Map![Python 2.7](https://img.shields.io/badge/python-2.7-blue.svg)
-
-
+# PokemonGo Map! ![python 2.7](https://img.shields.io/badge/python-2.7-blue.png "python 2.7")
```
Comment (jnovack) [inline README.md]: Why change from SVG to PNG?

**Comment by commanderjsoc** [comment]
For whatever reason, markdown doesn't like .svg and .json it only likes .png

**Comment by andrewda** [comment]
:+1:

File: README.md (hunk 1 above)
Comment (brycedorn) [inline README.md]: should revert to svg, it's about 1/3 the size of the png

Comment (commanderjsoc) [inline README.md]: Cant for what ever reason. Markdown wont display the img

Comment (brycedorn) [inline README.md]: wat, it shows for me
screen shot 2016-07-28 at 10 24 46 pm

Comment (commanderjsoc) [inline README.md]: Hmm... will retry tomorrow. Its getting late over here.

**Pull request closed without merging**